app/views/layouts/_header.html.erb

```diff
@@ -5,40 +5,58 @@
     <% end %>
   </div>
   <div id='user-info'>
-    <span id='logout_area'>
-      <span id='instructor_logged_in_as'>
+    <ul>
+      <li id="user-info-dropdown">
+        <div class = "dropdown">
+        <span id='instructor_logged_in_as'>
         <% if allowed_to?(:role_is_switched?) %>
           <strong><%= @real_user.display_name %></strong>
         <% end %>
-      </span>
-      <span id='user-id'>
-        <%= @current_user.display_name %>
-        ( <strong><%= @current_user.user_name %></strong> )
-      </span>
-      <% unless allowed_to?(:role_is_switched?) %>
-        <%= link_to t('menu.settings'),
-                    settings_users_path,
-                    class: 'button inline-button settings' %>
-      <% end %>
-      <% if @current_course && allowed_to?(:role_switch?, @current_course) %>
-        <%= link_to t('main.role_switch.switch_role'),
-                    role_switch_course_path(@current_course.id),
-                    remote: true,
-                    class: 'button inline-button switch-role' %>
-      <% end %>
+        </span>
+          <span id='user-id'>
+          <%= @current_user.display_name %>
+        </span>
 
-      <% if @current_course && allowed_to?(:clear_role_switch_session?, @current_course) %>
-        <%= link_to t('main.role_switch.clear_role_switch_session'),
-                    clear_role_switch_session_course_path(@current_course),
-                    class: 'button inline-button switch-role' %>
-      <% end %>
-
-      <%= link_to(t('main.log_out'),
-                  logout_main_index_path,
-                  id: 'logout_link',
-                  method: :post,
-                  class: 'button inline-button') %>
-    </span>
+          <ul>
+            <% unless allowed_to?(:role_is_switched?) %>
+              <li>
+                <%= link_to t('menu.settings'),
+                            settings_users_path %>
+              </li>
+            <% end %>
+            <% if @current_course && allowed_to?(:role_switch?, @current_course) %>
+              <li>
+                <%= link_to t('main.role_switch.switch_role'),
+                            role_switch_course_path(@current_course.id),
+                            remote: true %>
+              </li>
+            <% end %>
+            <% if @current_course && allowed_to?(:clear_role_switch_session?, @current_course) %>
+              <li>
+                <%= link_to t('main.role_switch.clear_role_switch_session'),
+                            clear_role_switch_session_course_path(@current_course) %>
+              </li>
+            <% end %>
+            <li>
+              <%= link_to t(:about_markus),
+                          about_main_index_path,
+                          remote: true %>
+            </li>
+            <li>
+            <span class="issues">
```

Comment:
I think you can remove the "span" part and the class, and just keep the `a` tag.

Reply:
Done